c/planarityTestGraphFunctionality.c

```diff
@@ -50,17 +50,22 @@ int TestGraphFunctionality(char *commandString, char *infileName, char *inputStr
 			else
 				Result = transformFile(theGraph, infileName);
 
-			if (Result == NOTOK) {
+			if (Result != OK) {
 				ErrorMessage("Unable to transform file.\n");
 			}
 			else
 			{
+				// Want to know whether the output is 0- or 1-based; will always be 
+				// 0-based for transformations of .g6 input
+				if (outputBase != NULL)
+					(*outputBase) = theGraph->internalFlags & FLAGS_ZEROBASEDIO;
```

Comment:
I think you need parens around the bitwise and, then use ternary conditional of ? 1 : 0.  This will give you a 1 regardless of which bit FLAGS_ZEROBASEDIO happens to be.